README.md

```diff
@@ -65,8 +65,8 @@ As for the rest, the interface should be pretty straightforward. If you have a q
    cd melonDS
    ```
 #### Dynamic builds (with DLLs)
-5. Install dependencies: `pacman -S mingw-w64-x86_64-{cmake,SDL2,toolchain,qt5-base,qt5-svg,qt5-multimedia,libslirp,libarchive}`
-6. Compile:
+1. Install dependencies: `pacman -S mingw-w64-x86_64-{cmake,SDL2,toolchain,qt5-base,qt5-svg,qt5-multimedia,qt5-tools,libslirp,libarchive}`
+2. Compile:
```

Comment:
There's no need to change the numbers here - they're labelled steps 5 and 6 because there are 4 steps before that you need to do, regardless of whether you are making a static or dynamic build.